modules/RTC/TraceablePeerConnection.js

```diff
@@ -1058,10 +1103,13 @@ const getters = {
         this.trace('getLocalDescription::preTransform', dumpSDP(desc));
 
         // if we're running on FF, transform to Plan B first.
-        if (RTCBrowserType.usesUnifiedPlan()) {
+        if (desc && RTCBrowserType.usesUnifiedPlan()) {
```

Comment:
since this is a getter, i think there's a check where we call get to see if we have one already or not